controllers/volumegroup_controller.go

```diff
@@ -215,6 +219,42 @@ func makeVolumeGroupName(prefix string, volumeGroupUID string) (string, error) {
 	return fmt.Sprintf("%s-%s", prefix, volumeGroupUID), nil
 }
 
+func (r *VolumeGroupReconciler) removeVolumesFromVG(logger logr.Logger, vg *volumegroupv1.VolumeGroup) error {
+	pvcList, err := utils.GetPVCList(logger, r.Client, r.DriverConfig.DriverName)
```

Comment:
if pvclist of vg is empty, should we still do all the checks?

Reply:
if the `pvcList` is empty, so it won't get into the for loop, which checks do you mean?

Reply:
I mean the `vg.pvcList`, not the one in line 223.
the checks in line 229.
if the `vg.pvcList` is empty, should we do anything here? is there anything to remove?

Reply:
got you

Reply:
done, handled it

Reply:
Tnx